packages/itwinui-react/src/core/index.ts

```diff
@@ -295,6 +295,22 @@ export type { TextareaProps } from './Textarea/index.js';
 export { Tile } from './Tile/index.js';
 export type { TileProps } from './Tile/index.js';
 
+export { TileNew } from './Tile/index.js';
+export type { TileNewProps } from './Tile/index.js';
```

Comment:
All of these types can go under the same set of braces

Reply:
I think I can't because exporting type requires `export type` ?
![image](https://github.com/iTwin/iTwinUI/assets/22462084/ad8f9702-b1be-4ed4-9be6-493ec8595116)

Unless you mean for TileNew stuff to go inside the same braces as original `Tile` right above, then I plan to rename `TileNew` to `Tile` soon and remove one of the set :d

Reply:
![image](https://github.com/iTwin/iTwinUI/assets/78564221/b3172824-8d05-42b3-9321-7b4cec076aec)
I was referring to all of these types to be in the same braces since they're all types and exported from the same file

Reply:
Ah yea that's right. Updated

Reply:
Closing since types do not have to be exported anymore